docs/docker/docker-troubleshooting.md

```diff
@@ -134,5 +134,14 @@ Start docker-sync so your docker-sync containers and volumes will be recreated:
 docker-sync start
 ```
 
+## Application is slow on Mac
+We focus on enhancing performance of application on all platforms.
```

Comment:
enhancing performance of application =>  enhancing the performance of the application